JOSESwift/Sources/CryptoImplementation/EC.swift

```diff
@@ -95,33 +95,46 @@ public enum ECCurveType: String, Codable {
     }
 }
 
-fileprivate extension SignatureAlgorithm {
-    typealias DigestFunction = (
-            UnsafeRawPointer?,
-            UInt32,
-            UnsafeMutablePointer<UInt8>?
-    ) -> UnsafeMutablePointer<UInt8>?
-
-    func createDigest(input: Data) throws -> [UInt8] {
-        guard
-                let computedDigestLength = digestLength,
-                let computedDigestFunction = digestFunction
-                else {
-            throw ECError.invalidCurveDigestAlgorithm
+// converting integers to and from DER encoded ASN.1 as described here:
```

Comment:
It would be great if you could add a short explanation of why exactly this padding is needed. Similar to your explanation for https://github.com/ethanhuang13/CupertinoJWT/issues/12 would be great. 👍

Reply:
Done.